### PR TITLE
StatsDOutputWriter should send metric with "Number" datat type as Counter metric to StatsD server

### LINKED ISSUE #2: StatsDOutputWriter should send "Number" as counter metrics
Currently StatsDOutputWriter sends all the (numbers, strings, etc) metrics as "Counter" to server. The server complains about that.

### PR DESCRIPTION
**Pull request by maheshkelkar**
Fixes #2

### CODE REVIEW

File: src/test/java/org/jmxtrans/agent/StatsDOutputWriterIntegrationTest.java [hunk 1]
```diff
@@ -26,6 +24,5 @@ public void test() throws IOException {
                 writer.writeQueryResult("jmxtrans-agent-test-metric-" + metricIndex, "counter", Integer.valueOf(10 * measureIndex + metricIndex));
```
Comment (GiriDandu) [inline src/test/java/org/jmxtrans/agent/StatsDOutputWriterIntegrationTest.java]: Where is this verified ? I mean output of StatsDOutputWriter

File: src/test/java/org/jmxtrans/agent/StatsDOutputWriterIntegrationTest.java [hunk 2]
```diff
@@ -13,7 +12,6 @@
 public class StatsDOutputWriterIntegrationTest {
 
     @Test
-    @Ignore
     public void test() throws IOException {
```
Comment (GiriDandu) [inline src/test/java/org/jmxtrans/agent/StatsDOutputWriterIntegrationTest.java:19]: Can we add non number metric value test case and make sure we don't write to StatsDOutuputWriter

Comment (maheshkelkar) [inline src/test/java/org/jmxtrans/agent/StatsDOutputWriterIntegrationTest.java:19]: done

File: src/test/java/org/jmxtrans/agent/StatsDOutputWriterIntegrationTest.java (hunk 1 above)
Comment (maheshkelkar) [inline src/test/java/org/jmxtrans/agent/StatsDOutputWriterIntegrationTest.java]: done

**Comment by maheshkelkar** [comment]
@GiriDandu I have taken care of the comments, please review.

**Comment by GiriDandu** [comment]
:+1:  merge away

**Pull request merged**